src/navigation/stack/navigationStack.js

```diff
@@ -11,11 +11,15 @@ import mainBucketNavigation from './mainBucketStack';
 import addContact from '../../screens/main/menu/contacts/add/AddContact';
 import contacts from '../../screens/main/menu/contacts/index';
 import ContactAddresses from '../../screens/main/menu/contacts/SelectedContact';
+import Splash from '../../screens/Splash'
 /**
  * Constant contains all the screens that can be navigated to using the
  * navigate method from any class
  */
 const navigator = StackNavigator({
+  splash: {
+    screen: Splash
+  },
```

Comment:
whats the purpose of this?

Reply:
This screen navigates to the portfolio page if a user already created a wallet or it navigates to the terms screen if the user didn't create a wallet yet